### ISSUE TITLE
Simplecov not properly installed

### ISSUE THREAD

**Issue by austinzhang323** [ldanielw1/mSpray#15]

**Comment by austinzhang323** [comment]
Austin@Austins-MBP:~/Documents/QA2022/mSpray$ rspec selenium_trial.rb

An error occurred while loading ./selenium_trial.rb. - Did you mean?
                    rspec ./spec/selenium_trial.rb

Failure/Error: __send__(method, file)

LoadError:
  cannot load such file -- /Users/Austin/Documents/QA2022/mSpray/selenium_trial.rb
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:2115:in `load'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:2115:in `load_file_handling_errors'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:1615:in `block in load_spec_files'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:1613:in `each'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:1613:in `load_spec_files'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:102:in `setup'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:86:in `run'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:71:in `run'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:45:in `invoke'
# /Users/Austin/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/exe/rspec:4:in `<top (required)>'
# /Users/Austin/.rbenv/versions/3.0.1/bin/rspec:23:in `load'
# /Users/Austin/.rbenv/versions/3.0.1/bin/rspec:23:in `<main>'
# 
#   Showing full backtrace because every line was filtered out.
#   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
#   RSpec::Configuration#backtrace_inclusion_patterns for more information.
No examples found.


Finished in 0.00006 seconds (files took 0.53368 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples

Coverage report generated for RSpec to /Users/Austin/Documents/QA2022/mSpray/coverage. 0 / 0 LOC (100.0%) covered.
Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected

**Comment by austinzhang323** [comment]
Hi Taipei, I don't know what's happening haha please help!

**Comment by ldanielw1** [comment]
Yep - this issue is blocked by https://github.com/ldanielw1/mSpray/issues/4
